### ISSUE TITLE
&attributes() -> String index out of range when mixin called with ()()

### ISSUE THREAD

**Issue by 0x00** [neuland/jade4j#117]
```jade
mixin ap-form__unit(legend)
  fieldset.ap-form__unit(class="#{legend != '' ? 'ap-form__unit--with-legend' : ''}")&attributes(attributes)
    if legend
      legend.ap-form__legend= legend
    block


mixin ap-form-row
  .ap-form-row
    block


doctype html
html
  head
    title test
  body
    +ap-form__unit()(class="ap-form__unit--bspace", data-controller="sell-category-selection")
      +ap-form-row()()
        p bla bla
````

I expect this one:
```html
<!DOCTYPE html>
<html>
  <head>
    <title>test</title>
  </head>
  <body>
    <fieldset class="ap-form__unit ap-form__unit--with-legend ap-form__unit--bspace" data-controller="sell-category-selection">
      <div class="ap-form-row">
        <p>bla bla</p>
      </div>
    </fieldset>
  </body>
</html>
```

**Comment by chbloemer** [comment]
I clould reduce the error to:
```jade
mixin first()
    block

+first()()
    p bla bla
```

**Comment by chbloemer** [comment]
Added testcase issues/117.jade. Fixed in 1.0.6.
